github/test_download_s3.py

```diff
@@ -0,0 +1,72 @@
+"""
+make sure we can download a file on demand
+"""
+import resources
+import pyjq
+import pytest
+
+
```

Comment:
as discussed, might want to include a comment explaining that this is testing pytest-services stuff rather than github